rotkehlchen/constants/assets.py

```diff
@@ -39,6 +39,7 @@
 A_BEST = Asset('eip155:1/erc20:0x1B073382E63411E3BcfFE90aC1B9A43feFa1Ec6F')
 A_GTC = Asset('eip155:1/erc20:0xDe30da39c46104798bB5aA3fe8B9e0e1F348163F')
 A_DIVA = Asset('eip155:1/erc20:0xBFAbdE619ed5C4311811cF422562709710DB587d')
+A_XDAI = Asset('eip155:100/erc20:0x0000000000000000000000000000000000000000')
```

Comment:
I didn't find an external source pointing to the xdai address. From my search the zero address seems to be the one.

Reply:
Another thing  I just noticed. xDAI is not an erc20 token so it has EvmTokenKind UNKOWN. Should this be stored as `Asset('eip155:100/unknown:0x0000000000000000000000000000000000000000')`  using unknown instead of erc20 in the string?